dev/Dockerfile

```diff
@@ -4,6 +4,49 @@ FROM ${IMAGE_NAME}
 USER root
 RUN apt-get update -y && apt-get install -y git
 
+#RUN apt-get install -y software-properties-common
+#RUN apt-get update && apt-get install -y  openjdk-11-jdk
+#RUN add-apt-repository ppa:openjdk-r/ppa
+
+#RUN apt-get update && apt-get install -y  openjdk-8-jdk
+
+RUN apt-get update && apt-get install -y gnupg2
+RUN sudo apt install -y software-properties-common
+RUN apt-get update && apt-get install -y wget procps && wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -
+
+RUN sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
+
+RUN sudo apt-get update && sudo apt-get install -y adoptopenjdk-8-hotspot
+
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       build-essential \
+       libsasl2-2 \
+       libsasl2-dev \
+       libsasl2-modules;
+
+#ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-arm64/
+ENV JAVA_HOME=/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64/jre/
+
+
+# --------------------------------------------------------
+# SPARK
+# --------------------------------------------------------
+
+ENV SPARK_VERSION=spark-3.1.3-bin-hadoop3.2
```

Comment:
This will make our dev environment heavy. Lets remove the logic for spinning up of Spark cluster from the Dockerfile, after we figure out how to make connections to remote Spark clusters